Makefile

```diff
@@ -32,6 +33,8 @@ else
 debian := $(wildcard /etc/debian_version)
 ifneq ($(strip $(debian)),)
 PKG_INSTALL=apt-get install -y
+SET_PYTHON=rm /usr/bin/python && ln -s /usr/bin/python3 /usr/bin/python
```

Comment:
I think this is a pretty drastic solution to this problem. I don't think people will be happy when a package is doing that.

Reply:
I'll retest on Debian to see if this can be removed.

Reply:
Yes this can be removed.  But `make install` still fails due to https://github.com/SUSE/DeepSea/issues/1702

Reply:
I think there needs to be a different solution to solve the linking from python -> python3.

Unfortunately I'm not really firm when it comes to build processes :/ maybe @smithfarm can help us here?

Reply:
That step is not needed. I can confirm now that this was added while troubleshooting #1702 but should be removed because it does not fix #1702   
```
root@ubuntu1804:/tmp/deepsea/DeepSea-v0.9.22# python --version
Python 3.6.8
```

Reply:
I have updated PR for Ubuntu. Actually this may be fixes #1702 on Ubuntu.  I'll retest now.

Reply:
This Makefile works on Ubuntu 18.04 LTS now

Note It's important that correct salt (py3) repo is configured (to avoid #1702 issue) but that is outside scope of this PR.
```
vagrant@ubuntu1804:~$ cat /etc/apt/sources.list.d/saltstack.list 
deb http://repo.saltstack.com/py3/ubuntu/18.04/amd64/latest bionic main
``